docs/topics/getting-started-cd-osl.adoc

```diff
@@ -0,0 +1,97 @@
+= Continuous Delivery using {OpenShiftLocal}
+
+In addition to building on your local machine and pushing to a local or public cloud, you also have the ability to set up continuous delivery for a booster. Specifically you can use the {launcher} application to clone a new booster repository to your GitHub account and have it deployed to your {OpenShiftLocal}. 
+
+. link:{link-launcher-openshift-local-install-guide}#create-launcher-app[Install the {launcher}] application to your {OpenShiftLocal}.
+. Navigate to the {launcher} application on your {OpenShiftLocal} using your browser. You can also use the following command to open the {launcher} in your browser:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ minishift console
```

Comment:
This won't open the launcher in the browser, this will open the OpenShift console. From there, you can open the launcher, sure.